core/src/test/java/nu/yona/server/goals/service/ActivityCategoryServiceIntegrationTest.java

```diff
@@ -30,11 +29,11 @@
 
 import nu.yona.server.goals.entities.ActivityCategory;
 import nu.yona.server.goals.entities.ActivityCategoryRepository;
-import nu.yona.server.goals.service.ActivityCategoryServiceTests.TestConfiguration;
+import nu.yona.server.goals.service.ActivityCategoryServiceIntegrationTest.TestConfiguration;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = { TestConfiguration.class })
-public class ActivityCategoryServiceTests
+public class ActivityCategoryServiceIntegrationTest
 {
 	private Set<ActivityCategory> activityCategories = new HashSet<ActivityCategory>();
```

Comment:
Comment applies to the ``mockRepository`` below. Does it make sense to add add mock calls in the setup if you are autowiring the repository?